modules/swagger-codegen/src/main/resources/typescript-fetch/api.mustache

```diff
@@ -302,14 +302,48 @@ export const {{classname}}Factory = function (configuration?: Configuration, fet
     };
 };
 
+
+{{#withInterfaces}}
+/**
+ * {{classname}} - interface{{#description}}
+ * {{&description}}{{/description}}
+ * @export
+ * @interface {{classname}}
+ */
+export interface {{classname}}Interface {
+{{#operation}}
+    /**
+     * {{&notes}}
+     {{#summary}}
+     * @summary {{&summary}}
+     {{/summary}}
+     {{#allParams}}
+     * @param {{=<% %>=}}{<%%dataType%%>}<%={{ }}=%> {{^required}}[{{/required}}{{paramName}}{{^required}}]{{/required}} {{description}}
```

Comment:
maybe use triple curly braces, i.e. `{{{` and `}}}`, to prevent potential html encoding of e.g. `Array<number>` to `Array&lt;number&gt;` ?

Reply:
I did some research into this and it looks kind of ugly doing the triple brackets within Custom Delimiters. So you can also just do `{<%&datatype%>}` or `{{&` and `}}` to escape html, so will try implementing this using `&`